frontend/src/components/visual-editor/v2/Diagrams.tsx

```diff
@@ -174,10 +183,13 @@ const Diagrams = () => {
   }, []);
 
   useEffect(() => {
+    const filteredBlocks = blocks.filter(
+      (block) => block.category === selectedCategoryId,
+    );
```

Comment:
Please check my last comment to avoid these changes

Reply:
Thanks :pray: 
Can you please check again ?